src/config.js

```diff
@@ -28,8 +28,9 @@ const config = {
         useYou: true
       },
       realtimeActive: false,
+      omitCommentBubbles: true,
```

Comment:
What else about comments will we be adding? It'll be all in 'frontend preferences' field, and I don't want settings to be overly-nested thing if possible.